docs/populating-configuration.md

```diff
@@ -0,0 +1,353 @@
+> NOTE some step in this document might refer to Red Hat internal components
+  that you do not have access to
+
+# Populating Configuration
+The document describes how to prepare Fleet Manager to be able to start by
+populating its configurations.
+
+For convenience, the following Makefile target is provided:
+
+```
+make secrets/setup/empty
+```
+
+This target initializes empty files that are necessary for the service to boot.
+It can be used to quickly get the service running locally, although it does not
+provide a functional kas-fleet-manager.
+
+Follow the subsections to find information about how to configure these files properly.
+
+## Interacting with the Fleet Manager API
+
+See the [Interacting with Fleet Manager](interacting-with-fleet-manager.md)
+document.
+
+## Setting up OCM tokens
+
+The Fleet Manager itself requires the use of an OCM token so it can
```

Comment:
Yes, it would work with the temporary, non-offline tokens. But as you said, that token expires after 7-10m so it's not really usable when running kfm as you've to update it every few mins. It would be best to just ensure everyone is using their offline token here to avoid it. 

> Am I correct or I am confusing it with another scenario?

You're correct but you may also be remembering that offline tokens cannot be used for interacting with the kfm api. It has to be the generated non-offline token as the offline token don't have all the claims kfm needs.

Reply:
> You're correct but you may also be remembering that offline tokens cannot be used for interacting with the kfm api. It has to be the generated non-offline token as the offline token don't have all the claims kfm needs.

This case is already documented properly in the "Interacting with Fleet Manager" document

Reply:
> Yes, it would work with the temporary, non-offline tokens. But as you said, that token expires after 7-10m so it's not really usable when running kfm as you've to update it every few mins. It would be best to just ensure everyone is using their offline token here to avoid it.

That seems reasonable. I updated this section to refer to it as an OCM offline token